.github/workflows/ppx-pr.yaml

```diff
@@ -0,0 +1,21 @@
+name: PR Build
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build-ppx:
+    name: PR Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Esy
+      run: yarn global add esy
```

Comment:
I don't think this should be necessary since `esy` is included in the `devDependencies`? Nothing necessarily wrong with installing it globally but doing both could lead to conflicts down the line (and probably pinning the `esy` version is the safest bet vs always using the latest version as done here)